inst/rmd/multiplex_qc.Rmd

```diff
@@ -0,0 +1,91 @@
+  )|>
+  t()
+
+knitr::kable(hto_information, align = 'r') |>
+  kableExtra::kable_styling(bootstrap_options = "striped",
+                            full_width = FALSE,
+                            position = "left") |>
+  kableExtra::column_spec(2, monospace = TRUE)
+```
+
+## Hashtag Oligos (HTOs) Statistics
+
+```{r}
+hto_tags <- as.data.frame(rowData(multiplex_exp)) |>
+  tibble::rownames_to_column("Hashtag Oligo (HTO)") |>
+  arrange(desc(mean)) |>
+  select("Hashtag Oligo (HTO)",
+         "Mean UMI count per cell" = mean,
+         "Percent of cells detected" = detected)
+
+knitr::kable(hto_tags, digits = 2) |>
+  kableExtra::kable_styling(bootstrap_options = c("striped", "condensed"),
+                            full_width = FALSE,
+                            position = "left",
+                            ) |>
+  kableExtra::column_spec(2:3, monospace = TRUE) 
+```
+## Demultiplexing Cell Calls 
+
+Demultiplexing was performed using both [DropletUtils::hashedDrops](https://rdrr.io/github/MarioniLab/DropletUtils/man/hashedDrops.html) and [Seurat::HTOdemux](https://rdrr.io/github/satijalab/seurat/man/HTODemux.html) only on the _filtered_ cells, using default parameters for each.
+
+For multiplex libraries where bulk RNA-seq data is available for the individual samples, we also performed demultiplexing analysis using genotype data following the methods described in [Weber _et al._ (2021)](https://doi.org/10.1093/gigascience/giab062). 
+The genetic demultiplexing results are reported under the `vireo` column in the below table.   
+
+**Note:** We have reported the demultiplexed sample calls for each of the above mentioned algorithms, but the multiplexed library has not been separated into individual samples. 
+
+```{r}
+# grab columns that have demuxing results as not all methods could be present
+# e.g. vireo is only used if a matching bulk RNA seq library is there, 
+# otherwise those calls will not be present
+demux_methods <- c("hashedDrops_sampleid", "HTODemux_sampleid", "vireo_sampleid")
+demux_columns <- colnames(colData(filtered_sce)) %in% demux_methods
+
+# create a table summarizing demuxing calls for each method used 
+demux_calls <- as.data.frame(colData(filtered_sce)[, demux_columns]) |>
+  # remove _sampleid at the end of the column names
+  dplyr::rename_with(~stringr::str_remove(., "_sampleid")) |>
+  tidyr::pivot_longer(cols = everything(),
+                      names_to = "demux_method",
+                      values_to = "Sample") |>
+  dplyr::count(Sample, demux_method) |>
+  tidyr::pivot_wider(names_from = demux_method,
+                     values_from = n)
+  
+  
+knitr::kable(demux_calls, digits = 2) |>
+  kableExtra::kable_styling(bootstrap_options = c("striped", "condensed"),
+                            full_width = FALSE,
+                            position = "left",
+                            ) |>
+  kableExtra::column_spec(2:3, monospace = TRUE) 
```

Comment:
Looking at the results, the last column was not monospaced, but rather than just changing 3 to 4 I made it a bit more flexible:
```suggestion
  kableExtra::column_spec(2:ncol(demux_calls), monospace = TRUE) 
```